release-pr/src/neon_release_pr/git.py

```diff
@@ -169,18 +169,7 @@ def base_branch_name() -> str:
 
 
 def release_branch_name_override() -> str | None:
-    overrides = {
-        "neondatabase/neon": {
-            "storage": "release",
-        },
-        "databricks-eng/hadron": {
-            "storage": "neon-release-storage",
-            "compute": "neon-release-compute",
-            "proxy": "neon-release-proxy",
-            "hcm": "release-compute",
-            "hadron": "release",
-        },
-    }
+    overrides = {}
```

Comment:
nit: does it make sense to have the rest of the function, as it will always return `None` in
```
if repo not in overrides:
...
```
imo with empty `overrides` info message below will also be confusing, and we may adjust logging as well